spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  describe 'content of user index page' do
+    let(:user1) { User.create(name: 'Juwairiyya', photo_link: 'photo_url', bio: 'text', posts_counter: 0) }
+
+    before(:each) do
+      user1.posts.create(title: 'Post Title', text: 'Post Content')
+      visit "/users/#{user1.id}"
+    end
```

Comment:
- [ ] For the "user show page:", i notice you're missing two tests. Kindly add the upbased on requirements given, so we can verify the flow of the app 👏 

- I can see the user's first 3 posts.
- When I click a user's post, it redirects me to that post's show page.

### Missing tests:
![image](https://github.com/Zuwairanajma/Blog_App/assets/50990925/c07e1dac-b910-43b8-a8af-86b3c5830979)